utils/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+#[macro_use]
+extern crate vmm_sys_util;
+use std::os::raw::c_int;
+pub use vmm_sys_util::{errno, eventfd, fam, ioctl, signal, tempdir, tempfile, terminal};
+
+/// Wrapper to interpret syscall exit codes and provide a rustacean `io::Result`
```

Comment:
Any reason to keep this functionality in the root of the module?

Reply:
This is exposed in (sys_util/srs/lib.rs)[https://github.com/firecracker-microvm/firecracker/blob/master/sys_util/src/lib.rs#L28] right now. As a consequence, I add it also into `utils/src/lib.rs`.

Reply:
Can we add an issue to move this?

Reply:
An issue to move it in a specific place, or an issue opened for discussions?
For this PR, is it OK for you if I let this functionality here?

Reply:
We should create an issue that basically says move the floating code from utils/lib.rs that treats syscall exit codes to a separate module. It's okay to leave the functionality as is in this PR.

Reply:
Sure, thanks. I will open an issue.